test/compiler.test.js

```diff
@@ -1564,6 +1564,21 @@ describe('compiler', function() {
         .to.throw(/path-does-not-exist/);
     });
 
+    it(
+      'does not fail when an optional middleware cannot be resolved',
+      function() {
```

Comment:
jscs was complaining about the 80 characters things. Is the 80 characters limit an intentional choice?